src/config/mod.rs

```diff
@@ -69,6 +72,8 @@ create_config! {
     format_macro_matchers: bool, false, false,
         "Format the metavariable matching patterns in macros";
     format_macro_bodies: bool, true, false, "Format the bodies of macros";
+    skip_macro_names: MacroNames, MacroNames::default(), false,
+        "Skip formatting the bodies of macros invoked with the following names.";
```

Comment:
Does this option prevent formatting of macro definition bodies and macro calls or just macro calls? Based on the original issue (#5324) and the test examples I would expect that this just prevents macro calls from being formatted. Maybe we should update the description to make that more clear.

Reply:
Happy to update to be clearer - I also think the name of the option itself could be clearer, after realising the potential sources of confusion.

Potential options:
- skip_macro_invocations
- skip_macro_calls

I'd personally vote for *skip_macro_invocations* as it's more explicit, despite being longer.

Reply:
If you're leaning towards `skip_macro_invocations` then let's move forward with that one. I also wonder if we could update the help text here and in the Configurations.md file to be something like:

> rustfmt will skip formatting macro invocations for macros who's names appear in this list. set this value to "*" to prevent rustfmt from formatting all macros.
>
> Note: This option does not control formatting macro definitions.

Maybe just include the note in the Configurations.md file?